### ISSUE TITLE
"add competitor" search should filter out the # in a hashtag

### ISSUE THREAD

**Issue by bgirardot** [MissingMaps/missingmaps.github.io#89]
When talking about hashtags, people are going to put in the # symbol in and if it is not used in the search, it should be auto filtered out.

**Comment by kamicut** [comment]
How is this handled now @mojodna? Is this is till a bug?

**Comment by mojodna** [comment]
Yes. Including the `#` passes it through to the API, where `#`s are not present.
